lib/internal/Magento/Framework/View/Result/Layout.php

```diff
@@ -7,6 +7,7 @@
 namespace Magento\Framework\View\Result;
 
 use Magento\Framework;
+use Magento\Framework\App\Response\HttpInterface as HttpResponseInterface;
```

Comment:
Unfortunately not, since the signature of `\Magento\Framework\View\Result\Layout::renderResult()` is inherited from `\Magento\Framework\Controller\ResultInterface`.